.github/workflows/pr-check.yml

```diff
@@ -0,0 +1,17 @@
+name: Test greentea-client
+
+on: [push, pull_request]
+
+jobs:
+  test-app:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Install CMake and Ninja
+      uses: lukka/get-cmake@latest
```

Comment:
It would be nice to also install and run astyle, similar to we do in mbed-os:
https://github.com/ARMmbed/mbed-os/blob/7279ae2cd09257f5d8cc93dccfcc64a3102030b7/.travis.yml#L136-L140

We can copy .astylerc from mbed-os.

Reply:
Good idea. I'll add the astyle checks.